src/FileProvenanceElastic.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+ * This file is part of ePipe
+ * Copyright (C) 2019, Logical Clocks AB. All rights reserved
+ *
+ * ePipe is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU Affero General Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later version.
+ *
+ * ePipe is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License along with this program.
+ * If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <FileProvenanceConstants.h>
+#include "FileProvenanceElastic.h"
+
+FileProvenanceElastic::FileProvenanceElastic(std::string elastic_addr, std::string index,
+        int time_to_wait_before_inserting, int bulk_size, const bool stats, SConn conn) :
+ElasticSearchWithMetrics(elastic_addr, time_to_wait_before_inserting, bulk_size, stats),
+mIndex(index), mConn(conn) {}
+
+const FileProvenanceLogTable::FileProvLogHandler* FileProvenanceElastic::getLogHandler(const LogHandler* log) {
```

Comment:
same as before, do you use this method, you shouldn't have it here

Reply:
done